### PR TITLE
(DEV-3736) - Adds smooth animation to the add prayer card and menu

### PR DESCRIPTION
**Pull request by IsaacHardy**
### What does this PR do, or why is it needed?
Adds animation to add prayer card and menu. This adds the polish needed for a professional application.

### How do I test this PR?
1. Open up the app
2. Make sure animations are working as intended.
3. Check tests.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-XXX](#url)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by cdanwards** [comment]
@IsaacHardy Is there a reason that the DevotionalContentItem, Locations, and UserAvatarHeaderConnected tests were changed?

**Comment by IsaacHardy** [comment]
No idea how those changed. I'll look into it after lunch.

**Comment by IsaacHardy** [comment]
@cdanwards @richarddubay This is good to go now.

File: packages/newspringchurchapp/src/prayer/index.js [hunk 1]
```diff
@@ -241,63 +253,82 @@ class PrayerMenu extends PureComponent {
...
+                description={
+                  'Take a moment to send a prayer request that your NewSpring Church family can pray for.'
+                }
+                {...this.props}
+              />
+            </StyledAddPrayerContainer>
+          ) : (
+            <StyledContainer>
+              <NSIcon
+                onPress={() => {
```
Comment (richarddubay) [inline packages/newspringchurchapp/src/prayer/index.js:285]: If I press the arrow and not the button link, bad things happen.
Screenshot 2019-05-09 15 23 51

**Review by richarddubay — CHANGES_REQUESTED** [review]

File: packages/newspringchurchapp/src/prayer/index.js (hunk 1 above)
Comment (IsaacHardy) [reply packages/newspringchurchapp/src/prayer/index.js:285]: Good catch. Didn't even think about that.

Comment (IsaacHardy) [reply packages/newspringchurchapp/src/prayer/index.js:285]: @richarddubay Fixed.

**Review by richarddubay — APPROVED** [review]
Look good @IsaacHardy!
